lib/jsdom/living/helpers/svg/render.js

```diff
@@ -0,0 +1,44 @@
+"use strict";
+const { SVG_NS } = require("../namespaces");
+
+// https://svgwg.org/svg2-draft/render.html#TermNeverRenderedElement
+const neverRenderedElements = new Set([
+  "clipPath",
+  "defs",
+  "desc",
+  "linearGradient",
+  "marker",
+  "mask",
+  "metadata",
+  "pattern",
+  "radialGradient",
+  "script",
+  "style",
+  "title",
+  "symbol"
+]);
+
+// https://svgwg.org/svg2-draft/render.html#Rendered-vs-NonRendered
+exports.isRenderedElement = function isRenderedElement(elImpl) {
+  if (neverRenderedElements.has(elImpl._localName)) {
+    return false;
+  }
+
+  if (elImpl.getAttributeNS(null, "display") === "none") {
+    return false;
+  }
+
+  if (!elImpl._attached) {
+    return false;
+  }
+
+  if (!elImpl.parentNode || elImpl.parentNode._namespaceURI !== SVG_NS) {
```

Comment:
I think elements with no parent should not be rendered, probably? This might be redundant with isConnected; I'll try and see what happens...